plan9/README.Plan9

```diff
@@ -0,0 +1,32 @@
+This is William McBrine PDCurses 3.4
```

Comment:
Hm, in this case it may be better placed as a pull request there?
I guess that's where you started, but William's upstream is at 3.9 and this one is quite different in some places.

In general we normally don't refer to a specific version or maintainer within a port.

Reply:
The license is identical to the main one, I suggest to remove this file and have a look at another port's README like https://github.com/Bill-Gray/PDCurses/tree/master/dosvga how to mention additional copyrights and distribution status instead of a new file.

Reply:
This is indeed a rest from the old updated port I did in 2015. I should remove that line. Thanks.